src/textbooks/textbook-card/TextbooksCard.test.jsx

```diff
@@ -35,4 +69,83 @@ describe('<TextbookCard />', () => {
       expect(getByText(url)).toBeInTheDocument();
     });
   });
+
+  it('render edit TextbookForm after clicking on edit button', async () => {
+    const { getByTestId, queryByTestId } = renderComponent();
+
+    const editButton = getByTestId('textbook-edit-button');
+    userEvent.click(editButton);
+
+    await waitFor(() => {
+      expect(getByTestId('textbook-form')).toBeInTheDocument();
+      expect(queryByTestId('textbook-card')).not.toBeInTheDocument();
+    });
+  });
+
+  it('close edit TextbookForm after clicking on cancel button', async () => {
```

Comment:
[nit]: I ran this test without the `waitFor` wrappers and it ran successfully

Reply:
Fixed